lib/Elastica/Cluster/Settings.php

```diff
@@ -53,15 +53,13 @@ public function getPersistent($setting = '')
         $data = $this->get();
         $settings = $data['persistent'];
 
-        if (!empty($setting)) {
-            if (isset($settings[$setting])) {
-                return $settings[$setting];
-            } else {
-                return;
-            }
+        if (empty($setting)) {
+            return $settings;
         }
 
-        return $settings;
+        if (isset($settings[$setting])) {
+            return $settings[$setting];
+        }
     }
```

Comment:
We should probably add here `return null` so it is clear nothing is returned if non matches. Yes, that was also missing before. Outcome will be the same, but I think it is nice for readability.